### PR TITLE
IN-141: Added standard labels to all yaml files.

### PR DESCRIPTION
**Pull request by michaelamiethyst**

### CODE REVIEW

File: charts/eventrouter/templates/configmap.yaml [hunk 1]
```diff
@@ -13,3 +13,10 @@ metadata:
     chart: {{ template "namewithversion" . }}
     release: "{{.Release.Name}}"
     heritage: "{{.Release.Service}}"
+    app.kubernetes.io/name: {{ template "name" . }}
```
Comment (mikeln) [inline charts/eventrouter/templates/configmap.yaml]: this chart has no template "name"   (see _helpers for the correct templates).
I think it is probably "eventrouter.name"  but the question is what this name should really be...
a) just the name, b) name and version, c) name and namespace d) name and chart version , etc...
I suspect is is just name.

File: charts/eventrouter/templates/configmap.yaml [hunk 2]
```diff
@@ -13,3 +13,10 @@ metadata:
...
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/eventrouter/templates/configmap.yaml]: this should use template "namewithversion" or whatever the correct one is in _helpers

File: charts/eventrouter/templates/deployment.yaml [hunk 3]
```diff
@@ -8,6 +8,13 @@ metadata:
     chart: {{ template "namewithversion" . }}
     release: "{{.Release.Name}}"
     heritage: "{{.Release.Service}}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/eventrouter/templates/deployment.yaml]: see comments above

File: charts/eventrouter/templates/rbac-clusterrole.yaml [hunk 4]
```diff
@@ -8,6 +8,13 @@ metadata:
     chart: {{ template "namewithversion" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/eventrouter/templates/rbac-clusterrole.yaml]: see comments above

File: charts/eventrouter/templates/rbac-clusterrolebinding.yaml [hunk 5]
```diff
@@ -8,6 +8,13 @@ metadata:
     chart: {{ template "namewithversion" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/eventrouter/templates/rbac-clusterrolebinding.yaml]: see comments above

File: charts/eventrouter/templates/rbac-serviceaccount.yaml [hunk 6]
```diff
@@ -8,3 +8,10 @@ metadata:
     chart: {{ template "namewithversion" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/eventrouter/templates/rbac-serviceaccount.yaml]: see comments above

**Comment by mikeln** [comment]
_helper needs to be fixed for:

```
{{- define "namewithversion" -}}
{{- $name := default .Chart.Name .Values.nameOverride -}}
{{- printf "%s-%s" $name .Chart.Version | replace "+" "_" | trunc 63  -}}
{{- end -}}
```

**Comment by michaelamiethyst** [comment]
I've made all requested changes.  Thank you.

File: charts/eventrouter/templates/_helpers.tpl [hunk 7]
```diff
@@ -22,7 +22,7 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 */}}
 {{- define "namewithversion" -}}
 {{- $name := default .Chart.Name .Values.nameOverride -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 -}}
+{{- printf "%s-%s" $name .Chart.Version | replace "+" "_" | trunc 63  -}}
```
Comment (mikeln) [inline charts/eventrouter/templates/_helpers.tpl]: extra space after 63

**Pull request merged**